### ISSUE TITLE
unexpected behaviour list_policies()

### ISSUE THREAD

**Issue by exitcode0** [okta/okta-sdk-python#288]
the below code will result in `mfa_policies` being `None` despite the body containing a valid response from the API
unsure if this matters but my org is on OIE
```await client.list_policies({"type": "PASSWORD"})``` works as expected in this scenario

```py
#!/usr/bin/env python3
import time
import asyncio
from okta.client import Client as OktaClient

client = OktaClient()

async def main ():
    mfa_policies, response, error = await client.list_policies({"type": "MFA_ENROLL"})
    print(mfa_policies)

    print(response.get_body())

if __name__ == "__main__":
    start_time = time.time()
    loop = asyncio.get_event_loop()
    loop.run_until_complete(main())
    print("--- script took %s seconds ---" % (time.time() - start_time))

```

**Comment by csanders-git** [comment]
Doesn't seem to do with OIE, instead it seems that this policy type wasn't implemented by the maintainers for unknown reasons. MFA_ENROLL is still valid in OIE and I added a PR to provide basic support for this type. https://github.com/okta/okta-sdk-python/pull/316

**Comment by bretterer** [comment]
Resolved in 2.9.0
